docs/content/how-to-guides/configure-reactions/configure-gremlin-command-reaction/_index.md

```diff
@@ -3,8 +3,156 @@ type: "docs"
 title: "Configure a Gremlin Command Reaction"
 linkTitle: "Configure a Gremlin Command Reaction"
 weight: 70
-toc_hide: true
-hide_summary: true
 description: >
     Learn how to configure a Gremlin Command Reaction
----
\ No newline at end of file
+---
+
+The Drasi Gremlin Command Reaction allows you to use the Continuous Query results as parameters to commands that run against a Gremlin database. You can specify a Gremlin Command for each type of query result (Added, Updated or Deleted). The Gremlin Reaction is compatible with [Azure Cosmos DB for Gremlin](https://learn.microsoft.com/en-us/azure/cosmos-db/gremlin/introduction), [Apache TinkerPop Gremlin Servers](https://tinkerpop.apache.org/docs/3.4.4/reference/#gremlin-server), and [JanusGraph](https://janusgraph.org/).
+
+## Requirements
+On the computer from where you will create the Drasi Gremlin Command Reaction, you need to install the following software:
+- [Drasi CLI](/reference/command-line-interface/) 
+
+## Supported Gremlin Servers
+- [Azure Cosmos DB for Gremlin](https://learn.microsoft.com/en-us/azure/cosmos-db/gremlin/introduction) 
+- [Apache TinkerPop Gremlin Servers](https://tinkerpop.apache.org/docs/3.4.4/reference/#gremlin-server)
+- [JanusGraph](https://janusgraph.org/)
+
+## Creating the Reaction
+To create a Reaction, execute the `drasi apply` command as follows:
+
+```text
+drasi apply -f my-reaction.yaml
+```
+
+The `drasi apply` command is how you create all new Drasi resources (in this case a Reaction). The `-f` flag specifies that the definition of the new Reaction is contained in the referenced YAML file `my-reaction.yaml` and the `-n` flag specifies the Drasi namespace in which to create the Reaction (Drasi must already be installed in that namespace).
+
+## Reaction Definitions
+The YAML file passed to `drasi apply` can contain one or more Reaction definitions. Here is an example of a Drasi Gremlin Command Reaction definition:
+```yaml
+kind: Reaction
+apiVersion: v1
+name: gremlin-reaction
+spec:
+  kind: Gremlin
+  queries:
+    query1:
+  properties: 
+    addedResultCommand: g.addV('Hello-world-from').property('MessageId', @MessageId).property('name',@MessageFrom)
+    updatedResultCommand: g.V()
```

Comment:
How do you access the before and after values for an update?

Reply:
Will had some examples in the docs, but the basic logic is:

if the parameter begins with `before`, e.g. before.Id, we will use the Id from the result before the change. if it begins with `after`, e.g. After.id, we will use the result after the change. By default, the after result set is used